packages/analytics-js/src/components/utilities/consent.ts

```diff
@@ -0,0 +1,29 @@
+import { CookieConsentOptions } from '@rudderstack/analytics-js/state/types';
+
+/**
+ * A function to get the name of the consent manager with enabled true set in the load options
+ * @param cookieConsentOptions Input provided as load option
+ * @returns string|undefined
+ *
+ * Example input: {
+ *   oneTrust:{
+ *     enabled: true
+ *   }
+ * }
+ *
+ * Output: 'oneTrust'
+ */
+const getUserSelectedConsentManager = (
+  cookieConsentOptions: CookieConsentOptions,
+): string | undefined => {
+  if (
+    !cookieConsentOptions ||
+    typeof cookieConsentOptions !== 'object' ||
```

Comment:
This will allow array type. Instead, use the utility method here.

Reply:
Addressed